draft-ietf-acme-acme.md

```diff
@@ -1972,6 +1997,11 @@ domain by verifying that the resource was provisioned as expected.
 5. Verify that key authorization provided by the server matches the token for
    this challenge and the client's account key.
 
+The server MAY follow redirects when dereferencing the URI.
```

Comment:
This needs WG review.   Exempting servers from standard HTTP semantics is novel, and it has serious operational implications for servers, since there are lots of solutions for things like load-balanced servers that rely on redirects.